components/azure-iot/thirdpartynotice.txt

```diff
@@ -0,0 +1,18 @@
+﻿Third Party Notices for Azure IoT SDKs project
```

Comment:
Since this component is now not under Microsoft Github account, is this note needed anymore? The "third party" term is not clear here, since the application developers receives the SDK from Espressif, and https://github.com/ustccw/AzureESP32 is an Espressif project (hosted under employee's account). So that is not "third party" anymore.